Author: Helmut Grohne <helmut@subdivi.de>
Description: make the build system honor $CC to facilitate cross builds

Index: memtester-4.3.0/conf-cc
===================================================================
--- memtester-4.3.0.orig/conf-cc	2012-06-09 23:45:22.000000000 +0200
+++ memtester-4.3.0/conf-cc	2015-09-12 20:36:27.000000000 +0200
@@ -1,3 +1,3 @@
-cc -O2 -DPOSIX -D_POSIX_C_SOURCE=200809L -D_FILE_OFFSET_BITS=64 -DTEST_NARROW_WRITES -c
+$CC -O2 -DPOSIX -D_POSIX_C_SOURCE=200809L -D_FILE_OFFSET_BITS=64 -DTEST_NARROW_WRITES -c
 
 This will be used to compile .c files.
Index: memtester-4.3.0/conf-ld
===================================================================
--- memtester-4.3.0.orig/conf-ld	2012-06-09 23:45:22.000000000 +0200
+++ memtester-4.3.0/conf-ld	2015-09-12 20:36:33.000000000 +0200
@@ -1,3 +1,3 @@
-cc -s
+$CC -s
 
 This will be used to link .o files into an executable.
Index: memtester-4.3.0/Makefile
===================================================================
--- memtester-4.3.0.orig/Makefile	2015-09-12 20:01:06.000000000 +0200
+++ memtester-4.3.0/Makefile	2015-09-12 20:50:40.000000000 +0200
@@ -10,8 +10,6 @@
 # You don't need to edit these; change the contents of the conf-cc and conf-ld
 # files if you need to change the compile/link commands.  See the README for
 # more information.
-CC			= $(shell head -n 1 conf-cc)
-LD			= $(shell head -n 1 conf-ld)
 
 SOURCES		= memtester.c tests.c
 OBJECTS		= $(SOURCES:.c=.o)
